test/type_prop/dyn_slice.cpp

```diff
@@ -179,9 +179,8 @@ INSTANTIATE_TEST_CASE_P(
     type_prop,
     DeduceDynSliceTest,
     ::testing::Values(
```

Comment:
@jbobba The changes in this file LGTM at a glance, but could you double-check?

Reply:
Is it ready? Should I "resolve" this?:)

Reply:
I'll request review from @jbobba, we can resolve this though :)